internal/pkg/cli/env_init.go

```diff
@@ -61,6 +61,19 @@ type InitEnvOpts struct {
 	*GlobalOpts
 }
 
+// Validate returns an error if the values passed by the user are invalid.
+func (opts *InitEnvOpts) Validate() error {
```

Comment:
Shall we rename `opts` to `o`? Same for the other functions in this file.

Reply:
Done! Thanks for catching it :)